### DISCUSSION TITLE
How to expand the existing language?

### DISCUSSION THREAD

**Discussion by mantou132** [zed-industries/zed#20109] [Feature Requests]
I want to add a styled-component highlight support for TypeScript, code example:

```ts
styled`
  color: red;
`
```

I need to copy all the original language support of TypeScript?

I have checked https://zed.dev/docs/extensions/languages, and did not find the answer. If I missed the important part, please tell me, thx.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
